### PR TITLE
Release 2.1 - WA

### PR DESCRIPTION
**Pull request by fergusoshea**
### JIRA link (if applicable) ###
N/A

### Change description ###
This PR is to keep Release-2.1 branch up to date with Release-2.0

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by marcjudge — DISMISSED** [review]

File: definitions/json/RoleToAccessProfiles/RoleToAccessProfiles-WA-nonprod.json [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+    "AccessProfiles": "GS_profile,caseworker-employment-etjudge-scotland",
+    "ReadOnly": "N",
+    "Disabled": "F",
+    "CaseAccessCategories": ""
+  },
+  {
+    "RoleName": "senior-tribunal-caseworker",
+    "Authorisation": "",
+    "CaseTypeID": "ET_Scotland",
+    "AccessProfiles": "GS_profile,caseworker-employment-legalrep-solicitor",
```
Comment (hsjhita) [inline definitions/json/RoleToAccessProfiles/RoleToAccessProfiles-WA-nonprod.json:42]: is there a reason why this has changed from caseworker-employment same with the tribunal-caseworker below. even in the prod file, it's caseworker-employment
https://github.com/hmcts/et-ccd-definitions-scotland/pull/449/files#diff-0032f608fec1fcba4bace39cdd9561ce32454358d270d031cd41cbfbbd3ee636L237

File: definitions/json/EventToComplexTypes/EventToComplexTypes-WA-nonprod.json [hunk 2]
```diff
@@ -0,0 +1,312 @@
+[
+  {
+    "ID": "ReferralDetails",
+    "CaseEventID": "createReferral",
+    "CaseFieldID": "referralCollection",
+    "ListElementCode": "referCaseTo",
+    "EventElementLabel": " ",
+    "FieldDisplayOrder": 1,
+    "DisplayContext": "OPTIONAL",
+    "Publish": "Y"
```
[the remark this answers is not in the document]
Comment (fergusoshea) [reply definitions/json/EventToComplexTypes/EventToComplexTypes-WA-nonprod.json:10]: Hi @hsjhita 
These do exist in the EW configs:
[EventToComplexTypes-WA-nonprod.json](https://github.com/hmcts/et-ccd-definitions-englandwales/pull/543/files#diff-0e898370217a04d3f70a115bde19d817122478bfd7dbcdd1f8df21cd3f1cb96e)
Line 356

File: definitions/json/CaseEventToFields/CaseEventToFields.json [hunk 3]
```diff
@@ -3818,7 +3818,8 @@
     "PageDisplayOrder": 2,
     "PageFieldDisplayOrder": 9,
     "ShowSummaryChangeOption": "Y",
-    "PageColumnNumber": 1
+    "PageColumnNumber": 1,
+    "Publish": "Y"
```
[the remark this answers is not in the document]
Comment (fergusoshea) [reply definitions/json/CaseEventToFields/CaseEventToFields.json]: Done

File: definitions/json/CaseEventToFields/CaseEventToFields.json [hunk 4]
```diff
@@ -3872,7 +3874,8 @@
     "PageID": 3,
     "PageDisplayOrder": 3,
     "PageFieldDisplayOrder": 4,
-    "ShowSummaryChangeOption": "Y"
+    "ShowSummaryChangeOption": "Y",
+    "Publish": "Y"
```
[the remark this answers is not in the document]
Comment (fergusoshea) [reply definitions/json/CaseEventToFields/CaseEventToFields.json]: Done

File: definitions/json/RoleToAccessProfiles/RoleToAccessProfiles-WA-nonprod.json (hunk 1 above)
Comment (fergusoshea) [reply definitions/json/RoleToAccessProfiles/RoleToAccessProfiles-WA-nonprod.json]: Done

File: definitions/json/CaseEvent/CaseEvent-WA-nonprod.json [hunk 5]
```diff
@@ -0,0 +1,380 @@
...
+    "PostConditionState": "Accepted(preAcceptCase.caseAccepted=\"Yes\"):1;Rejected",
+    "EventEnablingCondition": "caseType !=\"Multiple\" AND preAcceptCase.caseAccepted !=\"Yes\" AND managingOffice !=\"Unassigned\"",
+    "SecurityClassification": "Public",
+    "ShowEventNotes": "N",
+    "ShowSummary": "N",
+    "Publish": "Y"
+  },
+  {
+    "CaseTypeID": "ET_Scotland",
+    "ID": "et3Response",
```
Comment (hsjhita) [inline definitions/json/CaseEvent/CaseEvent-WA-nonprod.json:95]: Just a similar comment as EngWales - why would we publish this event? This doesn't submit the ET3

**Review by hsjhita — APPROVED** [review]

File: definitions/json/CaseEvent/CaseEvent-WA-nonprod.json (hunk 5 above)
Comment (fergusoshea) [reply definitions/json/CaseEvent/CaseEvent-WA-nonprod.json:95]: As stated in EW, this event was the original event requested by the ticket.
We've received a subsequent CR to update it as you suggest here: https://tools.hmcts.net/jira/browse/RET-4516

**Review by addison-jephtah — APPROVED** [review]

**Pull request merged**